src/nvim/ex_docmd.h

```diff
@@ -19,7 +19,20 @@
 #define EXMODE_NORMAL           1
 #define EXMODE_VIM              2
 
+/// The scope of a command.
+///
+/// The higher a number, the deeper a scope.
+typedef enum {
+  kCdScopeEditor,  ///< Affects the entire instance of NeoVim.
+  kCdScopeTab,     ///< Affects on tab page.
+  kCdScopeWindow,  ///< Affects one window.
+} CdScope;
+
+/// Last `:cd` scopes defined.
+#define MAX_CD_SCOPE  kCdScopeWindow
+
 #ifdef INCLUDE_GENERATED_DECLARATIONS
 # include "ex_docmd.h.generated.h"
 #endif
+
```

Comment:
And do not add a newline. This section *must* be at the very end and missing newline is missing for this reason. This is also how it is documented in the style guide.